src/api/v1/event/index.js

```diff
@@ -130,4 +132,25 @@ router.route('/')
     }).catch(next);
   });
 
+/**
+ * Uploads a profile picture for a given event. Requires admin access
+ */
+router.post('/picture', Storage.bufferImageBlob(Media.mediaTypes.EVENT_COVER), async (req, res, next) => {
+  if (!req.user.isAdmin()) return next(new error.Forbidden());
+  if (!req.body.uuid) return next(new error.BadRequest('UUID must be provided in body'));
+  try {
+    const cover = await Storage
+      .uploadToS3(Media.mediaTypes.EVENT_COVER, req.file, req.body.uuid)
+      .then((data) => data.Location)
+      .catch((er) => next(er));
+    const updatedEvent = await Event
+      .findByUUID(req.body.uuid)
+      .then((event) => event.updateCover(cover))
+      .catch(next);
```

Comment:
This Promise is already in a `try-catch` so no need to individually catch errors.